djstripe/models/core.py

```diff
@@ -759,6 +759,12 @@ def add_card(self, source, set_default=True):
 
 		return new_payment_method.resolve()
 
+	def add_payment_method(self, payment_method):
+		from .payment_methods import PaymentMethod
+
+		stripe_customer = self.api_retrieve()
+		PaymentMethod.attach(payment_method, stripe_customer)
```

Comment:
Could do with test coverage of this

Reply:
I have added fixtures for payment method. Not sure if its cent percent correct way 171a500f5. I am still bit confused how to use it in tests though. That's why I haven't been able to add coverage for this.

Reply:
@tiwariayush I'd suggest using a test token instead of a test card (something like [this](https://runkit.com/lxmedina/5d40210dc16a86001358f2d6)); otherwise the tests would fail if the account has not activated [Process payments unsafely](https://dashboard.stripe.com/account/integration/settings). Other option would be to use test payment methods directly (3rd column [here](https://stripe.com/docs/testing#cards))

Reply:
@tiwariayush cool it looks like you're on the right track, I think it just needs a tweak for the magic id faking.   I'll try taking a look at that today.

Reply:
Fixture update done in ac4d439da0d43c07990b6808e2f36a465bbf38a6 , and I've added a test in 3c8bb780c99d009e5d9dad51e3af8f88ef7da6df

Reply:
also another test in 48993f2cb06e4a6d9906ce4fa65e1b695f16afe3